mibidata/tiff.py

```diff
@@ -64,16 +64,18 @@ def write(filename, image, sed=None, optical=None, ranges=None,
         multichannel: Boolean for whether to create a single multi-channel TIFF,
             or a folder of single-channel TIFFs. Defaults to True; if False,
             the sed and optical options are ignored.
-        write_float: If True, saves the image data as float32 values (for
-            opening properly in certain software such as Halo). Defaults to
-            False which will save the image data as uint16. Note: setting
-            write_float to True does not normalize or scale the data before
-            saving, however saves the integer counts as floating point numbers.
+        write_dtype: Forces the image data saved as either float or uint16. Can
+            specify `'float'` or `np.float` to force data to be saved as
+            floating point values or `'int'` or `np.uint16` to save as integer
+            values. Defaults to None, which saves data as its original type.
+            Note that forcing native float image data to uint16 could result
+            in a loss of precision as values are clipped.
```

Comment:
Our default TIFF generation will be float (in a pipeline outside of this library), and this PR enables that to persist as the default type since the TIFF writer will not alter that type unless specified by the user. 

That said, we are supporting float and uint16, not general int types.